partner_financial_risk/models/res_partner.py

```diff
@@ -0,0 +1,120 @@
+# -*- coding: utf-8 -*-
+# © 2016 Carlos Dauden <carlos.dauden@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from datetime import datetime
+from dateutil.relativedelta import relativedelta
+from openerp import api, fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = 'res.partner'
+
+    eval_risk_sale_order = fields.Boolean(
+        string="Eval Sale Orders", help="Compute in total risk")
+    max_risk_sale_order = fields.Monetary(
```

Comment:
It isn't documented, but :https://github.com/odoo/odoo/blob/e37ba6183269b0d075833efbb9bf26e99404247e/openerp/fields.py#L1131-L1130